fw/t/unit/test_http_parser_common.h

```diff
@@ -572,32 +554,39 @@ do_split_and_parse(int type, int chunk_mode)
  */
 static
 int __attribute__((unused))
-validate_data_fully_parsed(int type)
+validate_data_fully_parsed(int type, size_t sz_diff)
 {
 	TfwHttpMsg *hm = (type == FUZZ_REQ || type == FUZZ_REQ_H2)
 			? (TfwHttpMsg *)req
 			: (TfwHttpMsg *)resp;
 
+	size_t hm_exp_len = GET_FRAMES_TOTAL_SZ() - sz_diff;
```

Comment:
Just use `frames_total_sz` - there is no reason to mask it to a function. The same for `GET_FRAMES_MAX_SZ` and `GET_CURRENT_FRAME()`, the last one should be just
```C
#define CURRENT_FRAME     frames[frame_index]
```